packages/common-all/src/engine/EngineV3Base.ts

```diff
@@ -285,41 +296,39 @@ export abstract class EngineV3Base implements ReducedDEngine {
   async queryNotes(opts: QueryNotesOpts): Promise<QueryNotesResp> {
     // const ctx = "Engine:queryNotes";
     const { qs, vault, onlyDirectChildren, originalQS } = opts;
-
     // Need to ignore this because the engine stringifies this property, so the types are incorrect.
     // @ts-ignore
     if (vault?.selfContained === "true" || vault?.selfContained === "false")
       vault.selfContained = vault.selfContained === "true";
 
-    const items = await this.fuseEngine.queryNote({
-      qs,
+    const response = await this.queryStore.queryNotes(qs, {
       onlyDirectChildren,
       originalQS,
     });
-
+    if (response.isErr()) {
```

Comment:
On deciding which words/symbols to use to point to the underlying type I got used to use the term "result" to mean its a `Result` type or "resp" to mean to point to the `RespV3` type. Not wanting to do unnecessary bikeshedding here but wondering if we can converge on a name for this.

Reply:
got it -> i'm good with this and will start adopting. can you add it to the conventions doc if its not already there?